### PR TITLE
Adds POST /api/articles/:article_id/comments API

### PR DESCRIPTION
**Pull request by cdesale**
- Adds POST /api/articles/:article_id/comments API
- Adds necessary tests
- Adds endpoint.json doccumentation

### CODE REVIEW

File: errors.js [hunk 1]
```diff
@@ -15,5 +15,6 @@ exports.handleCustomErrors = (err, req, res, next) => {
 };
 
 exports.handleServerErrors = (err, req, res, next) => {
+  console.log(err);
```
Comment (nc-jonathan-mcguire) [inline errors.js]: remove logs before pushing

File: models/comment-model.js [hunk 2]
```diff
@@ -19,3 +19,29 @@ ORDER BY created_at DESC;`,
       return rows;
     });
 };
+exports.inserCommentsForArticleByArticleId = (article_id, username, body) => {
```
Comment (nc-jonathan-mcguire) [inline models/comment-model.js]: typo - but up to you if you want to name it like this

**Review by nc-jonathan-mcguire — APPROVED** [review]
Looks all good to me 👍

File: errors.js (hunk 1 above)
Comment (cdesale) [reply errors.js]: Done.

File: models/comment-model.js (hunk 2 above)
Comment (cdesale) [reply models/comment-model.js]: Done.

**Pull request merged**